Cartfile

```diff
@@ -1,5 +1,6 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 4.3
-github "mapbox/MapboxDirections.swift" ~> 0.23.0
+binary "https://s3.amazonaws.com/mapbox/mapbox-navigation-native/ios/builds/Mapbox-iOS-Navigation-Native.json"
+github "mapbox/MapboxDirections.swift" "return-response"
```

Comment:
Release a new version of this library before releasing the SDK.

Reply:
If MapboxDirections.swift is otherwise stable, can we knock this out earlier? Maybe @frederoni can do it while we're working on these other things in Pacific & Mountain time?